src/interfaces/assistants_web/src/components/Shared/Icon/Icon.tsx

```diff
@@ -74,11 +76,24 @@ export const Icon: React.FC<{
     'h-icon-lg w-icon-lg': size === 'lg',
     'h-icon-xl w-icon-xl': size === 'xl',
   });
-  return (
-    <div className={cn(sizeClass, className ?? 'fill-volcanic-100 dark:fill-marble-950')}>
+
+  const iconElement = (
+    <div className={cn(sizeClass, 'fill-volcanic-100 dark:fill-marble-950', className)}>
       {getIcon(name, kind)}
     </div>
   );
+  return withBg ? (
+    <div
+      className={cn(
+        'flex h-5 w-5 items-center justify-center rounded-sm bg-marble-800 dark:bg-volcanic-200',
+        bgClassName
+      )}
+    >
+      {iconElement}
+    </div>
```

Comment:
Feels like this should be another component and export from Icon. Maybe something like `IconWrapper` or something like that. Otherwise, we will keep increasing the component parameters, and it will be harder to maintain